kafka-rest/src/main/java/io/confluent/kafkarest/exceptions/KafkaRestExceptionMapper.java

```diff
@@ -19,7 +19,9 @@
 import io.confluent.rest.entities.ErrorMessage;
 import io.confluent.rest.exceptions.KafkaExceptionMapper;
 import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
 import org.apache.kafka.common.errors.SerializationException;
+import org.apache.kafka.common.errors.TopicDeletionDisabledException;
 
 public final class KafkaRestExceptionMapper extends KafkaExceptionMapper {
```

Comment:
I don't really know the motivation behind `KafkaRestExceptionMapper` vs `KafkaExceptionMapper` but I feel like `TopicDeletionDisabledException` which is a Kafka exception should be mapped in `KafkaExceptionMapper`.

@msn-tldr what do you think?

Reply:
Good point. I have moved it to `rest-utils`, and left the new test on this PR.